.github/workflows/ci.yml

```diff
@@ -116,7 +116,7 @@ jobs:
     - name: Build documentation
       run: |
         source activate pandas-dev
-        doc/make.py --warnings-are-errors | tee sphinx.log ; exit ${PIPESTATUS[0]}
+        doc/make.py --warnings-are-errors --num-jobs 1 | tee sphinx.log ; exit ${PIPESTATUS[0]}
```

Comment:
Nice, thanks!

Just a question about this - why not use auto here? Or is it to make sure the CI build stays the same, and can then be changed in a separate PR?

Reply:
I had looked at the build times [before](https://github.com/pandas-dev/pandas/runs/1755572250#step:6:1) and [after](https://github.com/pandas-dev/pandas/runs/1755788159#step:6:1) this line change; I must have gotten confused, and thus thought `auto` slowed down the build time in CI. Seems to have sped it up by 42 seconds. Not amazing, but 🤷 Removed.